src/Components/Aspire.Azure.Messaging.EventHubs/EventHubProducerClientComponent.cs

```diff
@@ -32,9 +32,21 @@ protected override void BindSettingsToConfiguration(AzureMessagingEventHubsProdu
         {
             EnsureConnectionStringOrNamespaceProvided(settings, connectionName, configurationSectionName);
 
-            return !string.IsNullOrEmpty(settings.ConnectionString) ?
-                new EventHubProducerClient(settings.ConnectionString, options) :
-                new EventHubProducerClient(settings.FullyQualifiedNamespace, settings.EventHubName, cred, options);
+            if (!string.IsNullOrEmpty(settings.ConnectionString))
```

Comment:
Maybe invert both negations for readability?

Reply:
Maybe we could use some pattern matching here instead of nested if clauses :)